charts/ziti-controller/templates/deployment.yaml

```diff
@@ -118,6 +119,10 @@ spec:
               {{- else }}
               value: {{ include "ziti-controller.fullname" . }}-client.{{ .Release.Namespace }}.svc.{{ .Values.ca.clusterDomain }}:{{ .Values.clientApi.advertisedPort }}
               {{- end }}
+            {{- range $key, $val := .Values.env }}
```

Comment:
I suggest positioning this range loop as the last, after the others in the list of vars. There's not a specific need for this right now, but it allows custom vars to act as overrides if needed without changing the chart template. That seems like the correct precedence. 

This env reference is correct after we fix the default value for controller env. It's currently nested incorrectly under image.